.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: false
+language: ruby
+cache:
+  directories:
+    - vendor/bundle
+    - .gradle
+rvm:
+  - jruby-1.7.25
+  - jruby-9.0.5.0
```

Comment:
Hmm, well, it runs in parallel so it shouldn't slow things down. I just figure that if the tests pass that's a bonus since we'll have to move to it at some point?

WDYT?